packages/expo-dev-menu/ios/DevMenuDevOptionsDelegate.swift

```diff
@@ -0,0 +1,68 @@
+// Copyright 2015-present 650 Industries. All rights reserved.
+
+import Foundation
+
+class DevMenuDevOptionsDelegate {
+  internal private(set) weak var bridge: RCTBridge?
+  internal private(set) weak var devSettings: RCTDevSettings?
+  
+  #if DEBUG
+  internal private(set) weak var perfMonitor: RCTPerfMonitor?
+  #endif
+  
+  internal init(forBridge bridge: RCTBridge) {
+    self.bridge = bridge
+    devSettings = bridge.module(forName: "DevSettings") as? RCTDevSettings
+    
+    #if DEBUG
+    perfMonitor = bridge.module(forName: "PerfMonitor") as? RCTPerfMonitor
```

Comment:
Will it cause any issues that this code (and everything inside of the `#if DEBUG` sections) will run in published/production apps loaded in DEBUG builds of the dev client?

Reply:
I believe this code will run fine as long we are in the debug build. However, these options won't be available in the published app - users can't trigger them. 

But to be sure, I'll test that before beta.